services/bitbucket/bitbucket-pull-request.service.js

```diff
@@ -47,19 +98,33 @@ function pullRequestClassGenerator(raw) {
     static get route() {
       return {
         base: `bitbucket/${routePrefix}`,
-        format: '([^/]+)/([^/]+)',
-        capture: ['user', 'repo'],
+        format: `(?:(http|https)/(.+)/)?([^/]+)/([^/]+)`,
```

Comment:
What do you think about limiting the Bitbucket Server support to `https`?

Reply:
While I agree everyone ***should*** be using `https` in this day and age I don't know that that's always going to be the case. I know my company still has **tons** of internal apps on `http` only. Do other on-prem services have this limitation?

Reply:
All good points. I'd suggest a middle path which makes us part of the solution: if there's a documented limitation or pattern of running _this_ service without https then let's support it. If not we could at least wait until someone complains…

Reply:
The default documentation for running BBS via [docker](https://hub.docker.com/r/atlassian/bitbucket-server/) and on a [linux server](https://confluence.atlassian.com/bitbucketserver/install-bitbucket-server-on-linux-from-an-archive-file-868977010.html)  both use `http` as the default scheme, with extra links for setting up `https`. While this is certainly not requiring the use of `http`, I feel we should still allow users to specify the scheme if other services do so. If we want to start locking down services to only support `https`, I feel that should be a separate issue.

Reply:
Okay, I'm fine with that.